decorator.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(name)
+    super()
+    @nameable = name
+  end
```

Comment:
- [ ] your decorator class is missing `attr_accessor` kindly add `attr_accessor :nameable` 🙏

Reply:
This has been revised.Thank you 😄